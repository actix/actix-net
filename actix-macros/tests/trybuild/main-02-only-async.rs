#[actix_rt::main]
fn main() {
    futures_util::future::ready(()).await
}
