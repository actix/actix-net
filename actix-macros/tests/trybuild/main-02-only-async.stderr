error: only async fn is supported
 --> $DIR/main-02-only-async.rs:2:1
  |
2 | fn main() {
  | ^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> $DIR/main-02-only-async.rs:1:1
  |
1 | / #[actix_rt::main]
2 | | fn main() {
3 | |     futures_util::future::ready(()).await
4 | | }
  | |_^ consider adding a `main` function to `$DIR/tests/trybuild/main-02-only-async.rs`
